### PR TITLE
Fill filter attributes

### PR DESCRIPTION
**Pull request by Kagamine**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ardalis is the list correct? I can fix the links to fix the build.

**Comment by ardalis** [comment]
I'm not sure it's an exhaustive list, but the ones listed are all built-in filters, yes.

File: aspnet/mvc/controllers/filters.rst [hunk 1]
```diff
@@ -94,9 +94,11 @@ Several of the filter interfaces have corresponding attributes that can be used
 Filter attributes:
 
 - :dn:cls:`~Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute` 
-- AuthorizationFilterAttribute
 - :dn:cls:`~Microsoft.AspNetCore.Mvc.Filters.ExceptionFilterAttribute` 
 - :dn:cls:`~Microsoft.AspNetCore.Mvc.Filters.ResultFilterAttribute` 
+- :dn:cls:`~Microsoft.AspNetCore.Mvc.Filters.FormatFilterAttribute`
```
Comment (Rick-Anderson) [inline aspnet/mvc/controllers/filters.rst]: Doesn't have filter in the FQN - bad link

**Comment by Kagamine** [comment]
Should I remove the format filter?

**Comment by Rick-Anderson** [comment]
Sorry, what I meant is the FQN is
dn:cls:~Microsoft.AspNetCore.Mvc.~~Filters~~.FormatFilterAttribute

check the others

**Pull request merged**